From 2ba41564cb2a2584abf47e01ade76b0f6c39db68 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:09 -0400
Subject: [scsi] qla2xxx: Replace a constant with a macro definition for host->canqueue assigmnment

Message-id: <1408391817-6389-22-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87925
O-Subject: [PATCH RHEL 7.1 e-stor 21/69] qla2xxx: Replace a constant with a macro definition for host->canqueue assigmnment.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Armen Baloyan <armen.baloyan@qlogic.com>

scsi commit id a4e04d9a67713e928227bf0adcbcb8e4dfa70997

Signed-off-by: Armen Baloyan <armen.baloyan@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_mr.h b/drivers/scsi/qla2xxx/qla_mr.h
index 1416e5e..6cd7072 100644
--- a/drivers/scsi/qla2xxx/qla_mr.h
+++ b/drivers/scsi/qla2xxx/qla_mr.h
@@ -545,4 +545,7 @@ struct mr_data_fx00 {
 
 #define QLAFX00_CRITEMP_THRSHLD  80 /* Celsius degrees */
 
+/* Max conncurrent IOs that can be queued */
+#define QLAFX00_MAX_CANQUEUE  1024
+
 #endif
diff --git a/drivers/scsi/qla2xxx/qla_os.c b/drivers/scsi/qla2xxx/qla_os.c
index 10bcf42..eadedec 100644
--- a/drivers/scsi/qla2xxx/qla_os.c
+++ b/drivers/scsi/qla2xxx/qla_os.c
@@ -2715,7 +2715,7 @@ que_init:
  }
 
  if (IS_QLAFX00(ha))
-  host->can_queue = 1024;
+  host->can_queue = QLAFX00_MAX_CANQUEUE;
  else
   host->can_queue = req->num_outstanding_cmds - 10;
 
-- 
1.7.1